everglades_dryrun_workflow.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#SBATCH --job-name=everglades_workflow 
+#SBATCH --mail-user=henrysenyondo@ufl.edu 
+#SBATCH --mail-type=FAIL
+#SBATCH --gpus=a100:4
+#SBATCH --cpus-per-task=5
+#SBATCH --mem=1000gb
+#SBATCH --time=180:00:00
```

Comment:
We should need much less time from this so suggest scaling back to a reasonable number to help with scheduling.

Reply:
I'll trim it to 72 hours, roughly the time elapsed for the last two runs of a clean pipeline, which was about 3 days. I'll further reduce it pending how playing with number of workers goes.

Reply:
This is a dry run file though. I assume the test setup is running much faster right?